src/directory/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Person from "../directory/singleUser.jsx";
+
+const axios = require("axios");
+
+class directory extends React.Component {
```

Comment:
@Alihaider168 we name components in TitleCase across our app so let's stick with that convention.